aws-powertools/powertools-lambda-typescript
# chore: Switch metrics to npm-shrinkwrap

## Description of your changes

Removed `package-lock.json` and generated `npm-shrinkwrap.json` which turns out is the same exact file, hence git tracks it as a rename.








### How to verify this change

Either of these two:
* Remove current `node_modules` (if any) and run `npm i` again, then run tests.
* Confirm that automated check that have run on this PR are successful.






### Related issues, RFCs


[#174](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/174)  

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist


---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
